contracts/test/veLON/vBalance.t.sol

```diff
@@ -53,4 +54,222 @@ contract TestVeLONDeposit is TestVeLON {
         total = total.add(expectedAliceInitvBal);
         assertEq(veLon.totalvBalance(), total);
     }
+
+    function testCreateLock_and_Withdraw_DifferentTime() public {
+        // alice and bob despoit for different amount and time
+        uint256 totalBalance = 0;
+        uint256 aliceStakeAmount = 1 * 365 days;
+        uint256 aliceTokenId = _stakeAndValidate(alice, aliceStakeAmount, 2 weeks);
+        uint256 aliceBalance = _initialvBalance(aliceStakeAmount, 2 weeks);
+        totalBalance = totalBalance.add(aliceBalance);
+        assertEq(veLon.vBalanceOf(aliceTokenId), aliceBalance);
+        assertEq(veLon.totalvBalance(), totalBalance);
+
+        uint256 bobStakeAmount = 5 * 365 days;
+        uint256 bobTokenId = _stakeAndValidate(bob, bobStakeAmount, 1 weeks);
+        uint256 bobBalance = _initialvBalance(bobStakeAmount, 1 weeks);
+        totalBalance = totalBalance.add(bobBalance);
+        assertEq(veLon.vBalanceOf(bobTokenId), bobBalance);
+        assertEq(veLon.totalvBalance(), totalBalance);
+
+        // check the totalSupply
+        assertEq(veLon.totalSupply(), 2);
+
+        // fast forward 1 weeks, bob's lock is expired
+        uint256 dt = 1 days;
+        vm.warp(block.timestamp + 7 * dt);
+        vm.roll(block.number + 1);
+        // sub Bob's all balance
+        totalBalance = totalBalance.sub(bobBalance);
+        // sub Alice's delined balance ()
+        totalBalance = totalBalance.sub((1 * 7 days));
+
+        // alice's balance
+        uint256 expectedAliceBalance = 1 weeks;
+        assertEq(veLon.vBalanceOf(aliceTokenId), expectedAliceBalance, "Alice's balance is not as expected");
+
+        vm.prank(bob);
+        veLon.withdraw(bobTokenId);
+        assertEq(veLon.totalvBalance(), totalBalance, "expectedBalance is not equal");
+    }
+
+    function testCreateLock_and_OneEarlyWithdraw() public {
+        // alice and bob despoit for different amount and time
```

Comment:
Both use the same amount and lock duration. Update comment or change testing code.

Reply:
Updated the comments for **same amount and time**